src/js/index.js

```diff
@@ -85,3 +85,69 @@ $(document).ready(function(){
   }).resize();
 
 }); //jQuery
+
+//smooth scrolling
+$(function() {
```

Comment:
Ah yes, thanks. I pulled most of that code from a codepen and it must have taken the spacing with it.